tox.ini

```diff
@@ -379,7 +388,9 @@ changedir=
   all_daemons: {toxinidir}/tests/functional/all_daemons{env:CONTAINER_DIR:}
   cluster: {toxinidir}/tests/functional/all_daemons{env:CONTAINER_DIR:}
   shrink_mon: {toxinidir}/tests/functional/shrink_mon{env:CONTAINER_DIR:}
+  shrink_mgn: {toxinidir}/tests/functional/shrink_mon{env:CONTAINER_DIR:}
```

Comment:
typo ?

Reply:
Weird. I wonder how tests passed locally and on CI. Besides, i don't see this mistake in my repo. Anyways, I'll push again.